proto/dydxprotocol/listing/genesis.proto

```diff
@@ -4,4 +4,8 @@ package dydxprotocol.listing;
 option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/listing/types";
 
 // GenesisState defines `x/listing`'s genesis state.
-message GenesisState {}
\ No newline at end of file
+message GenesisState {
+  // permissionless_listing_enabled defines whether permissionless listing is
+  // enabled.
+  bool permissionless_listing_enabled = 1;
```

Comment:
this is fine for now but i assume `x/listing` will have other parameters in the future? If so, we usually define a `Params` proto and have `GenesisState` wrap `Params` (e.g. [here](https://github.com/dydxprotocol/v4-chain/blob/main/proto/dydxprotocol/vault/genesis.proto#L10-L13)).

Reply:
I don't foresee this specific msg having more params. This msg is purely to enable/disable the PML feature. The actual market listing will be different